go.mod

```diff
@@ -9,47 +9,63 @@ replace github.com/docker/distribution => github.com/2opremio/distribution v0.0.
 // sops needs to update their deps ref: https://github.com/kubernetes/client-go/issues/628
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
 
+// transitive requirement from Helm Operator (to be removed on helm-op 1.0.0 GA)
+replace (
+	github.com/docker/docker => github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
```

Comment:
As long as Helm does not drop this replace, we will never be able to drop it ourselves, as far as I am aware.

Reply:
I can change it to `to be re-evaluated on helm-op 1.0.0 GA`?